sympy/physics/control/lti.py

```diff
@@ -249,6 +252,115 @@ def expand(self):
         """
         return TransferFunction(expand(self.num), expand(self.den), self.var)
 
+    def dcgain(self):
+        """
+        Computes the low-frequency gain of the frequency response in the
+        limit as frequency approaches to zero. For systems with pure
+        integrator terms, DC gain is infinite.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p, a, b
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> tf1 = TransferFunction(s + 3, s**2 - 9, s)
+        >>> tf1.dcgain()
+        -1/3
+        >>> tf2 = TransferFunction(p**2, p - 3 + p**3, p)
+        >>> tf2.dcgain()
+        0
+        >>> tf3 = TransferFunction(a*p**2 - b, s + b, s)
+        >>> tf3.dcgain()
+        (a*p**2 - b)/b
+
+        """
+        m = Mul(self.num, Pow(self.den, -1, evaluate=False), evaluate=False)
+        return limit(m, self.var, 0)
+
+    def pole(self, *, multiple=True):
```

Comment:
```suggestion
    def pole(self):
```
With the change to use `rootof`, there's little point in this any more.